content/faq/content.md

```diff
@@ -3,19 +3,19 @@ title: How does LBRY handle content and what am I allowed to upload?
 category: publisher
 ---
 
-This guide provides answers to questions regarding what you may upload to the LBRY app and network and how it differs from current centralized/status quo systems. To learn more about reporting infringing or illegal content and DMCA producedures, please see our [DMCA article](https://lbry.io/faq/dmca). 
+This guide provides answers to questions regarding what you may upload to the LBRY app and network and how it differs from current centralized/status quo systems. To learn more about reporting infringing or illegal content and DMCA procedures, please see our [DMCA article](https://lbry.io/faq/dmca). 
 
 ### What content can I legally upload to LBRY?
 
-You may upload content you created or own that does not infringe on the rights of others or is illegal, or other content you have permission to upload. This includes Public Domain content, Creative Commons or Open Source content shared according to the license requirements, and Copyrighted content where you have permission from the owner.
+You may upload content you created or own that does not infringe on the rights of others, is not illegal, or other content you have permission to upload. This includes Public Domain content, Creative Commons or Open Source content shared according to the license requirements, and Copyrighted content where you have permission from the owner.
```

Comment:
This needs another pass. Perhaps:
You may upload legal content that you own or have obtained permission for as long as it does not infringe on the rights of others.

Reply:
Amended.